types/maxmsp/index.d.ts

```diff
@@ -273,17 +281,17 @@ declare class Dict {
     /**
      * Set the value for a key to a specified value, creating heirarchy.
      */
-    replace(key: string, value: any[]): void;
+    replace(key: string, value: any): void;
 
     /**
      * Set the value for a key to a specified value.
      */
-    set(key: string, value: any[]): void;
+    set(key: string, value: any): void;
 
     /**
-     * Set the value for a key to dictionary content defined using JSON.
+     * Set the value for a key to dictionary content defined using JSON as a string.
      */
-    setparse(key: string, value: any[]): void;
+    setparse(key: string, value: string): void;
```

Comment:
`	d.setparse("meatloaf", '{ "price" : "priceless", "lyric" : "I would do anything for love but I won\'t eat that" }');
`